libraries/ESP8266WiFi/src/WiFiClient.h

```diff
@@ -30,6 +30,10 @@
 
 #define WIFICLIENT_MAX_PACKET_SIZE 1460
 
+#define TCP_DEFAULT_KEEPALIVE_IDLE_SEC          7200 // 2 hours
```

Comment:
Do #defines count against available heap, or is it better to make these static const and put them in flash?
(I'm thinking of the free heap project, every byte counts)

Reply:
unlike ```static const```, ```#define``` count for nothing in ram nor flash.